scipy/optimize/zeros.py

```diff
@@ -123,6 +123,35 @@ def newton(func, x0, fprime=None, args=(), tol=1.48e-8, maxiter=50,
     sign. The brentq algorithm is recommended for general use in one
     dimensional problems when such an interval has been found.
 
+    Examples
+    --------
+
+    ``fprime`` and ``fprime2`` not provided, use secant method
+    Only ``fprime`` provided, use Newton Raphson method
+    ``fprime2`` provided, ``fprime`` provided/not provided use parabolic
+    Halley's method
```

Comment:
These comments are now duplicate with the first paragraph of the docstring. May be useful to keep, but then split this up and put each method right above the example demonstrating it.